index.js

```diff
@@ -0,0 +1,15 @@
+const hamburger = document.querySelector('.hamburger');
+const Menu = document.querySelector('.menu');
+const navlinks = document.querySelectorAll('.nav-link');
+const closeBtn = document.querySelector('.close_btn');
+
+function open() {
+  hamburger.classList.toggle('active');
+  Menu.classList.toggle('active');
+}
+
+hamburger.addEventListener('click', open);
+closeBtn.addEventListener('click', open);
+navlinks.forEach((link) => {
+  link.addEventListener('click', open);
+});
```

Comment:
- [ ] [REQUIRED] Great implementation @Iptysam , however, there is a horizontal scroll bar on your mobile design which needs to be removed. Also, there is a white space tight at the top of the close button (X) that needs to be removed. Kindly fix these to get this milestone approved.
### Your Implemenatation
![Screenshot from 2022-09-06 18-56-39](https://user-images.githubusercontent.com/58692590/188706582-1893d95d-4ea0-4f09-963a-1b6f59f25bc7.png)

![Screenshot from 2022-09-06 18-56-39](https://user-images.githubusercontent.com/58692590/188706493-086b8d44-2292-4da8-8076-4b8a00ec04a2.png)